src/lib/converter/plugins/DynamicModulePlugin.ts

```diff
@@ -61,6 +61,9 @@ export class DynamicModulePlugin extends ConverterComponent {
             name = name.replace(/"/g, '');
             this.reflections.push(reflection);
             this.basePath.add(name);
+            if (context.getCompilerOptions().baseUrl) {
+                this.basePath.add(context.getCompilerOptions().baseUrl as string + '/file');
```

Comment:
This is needed because ```BasePath``` expects a file and then get the dirname, turning ```a/b/c``` into ```a/b``` this counteracts that.